src/content/getting started/understand-options-for-connectivity/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Expose your app to the internet
+description: Use the platform capabilities to expose your application to the internet.
+weight: 60
+last_review_date: 2024-04-05
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - How does basic connectivity work in the platform?
+  - What are the options for service connectivity in the platform?
+---
+
+<!--
+Explain how basic connectivity works (private vs public) through an example
+Topics
+- Private connectivity
+- Public connectivity
+- Default network policies
+- Ingress and egress
+
+Maybe example of an application that needs to connect to a database to show how it works
```

Comment:
Let's use the simplest possible example – a web server showing a simple "Hello world" page. I hope our hello-world app does just that (see [cluster-test-suites where we use hello-world](https://github.com/search?q=repo%3Agiantswarm%2Fcluster-test-suites%20hello-world&type=code)).